util/Migrator/DbScripts/2024-09-13_00_VerifiedOrgDomainSsoDetails.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR ALTER PROCEDURE [dbo].[OrganizationDomainSsoDetails_ReadByEmail]
```

Comment:
I do think the rename to add a `Verified` prefix to these operations is good. We should rename this stored procedure too, though, if we're going to do that. I think something like this would have to be done in an EDD style meaning:

1. Implement the change in a procedure with the old name
2. Duplicate the procedure under the new name
3. _Do not_ update the reference to the stored procedure in code
4. __RELEASE__
5. Update the reference in code
6. __RELEASE__
7. Drop the old procedure with the old name
8. __RELEASE__

This is pretty lame, but is a consequence of uptime/rollback-friendliness requirements for the database.